enhancements/testing/openshift-tests-extension.md

```diff
@@ -0,0 +1,722 @@
+                # disruption reporting.
+                "disruption": "1m",
+                # If, after applying the component configuration, openshift-tests
+                # should await cluster steady state before running configuration
+                # specific tests. If not specified, openshift-tests assumes
+                # the tests can be run immediately after applying the configuration
+                # profile.
+                "await": "steady-state"
+            },
+            
+            "resources": {
+              
+              # openshift-tests will only apply a single configuration
+              # profile to a component at a given time.
+              # However, inter-component conflicts might still occur.
+              # Configurations can request to be run in isolation
+              # from other component configurations. 
+              # If two component configurations have a conflict name in common, 
+              # test planning will ensure that there is no attempt to 
+              # apply those profiles simultaneously.
+              "isolation": {
+                "conflict":  [
+                  "gpu",
+                ]
+              },
+                
+              # Tests that require significant resources can identify
+              # that fact so that the planning algorithm will not
+              # overdraw on allocatable pod resources for a set of
+              # tests run in parallel.
+              "memory": "1Gi",
+              
+              # If a duration can be roughly predicted, inclusion of this
+              # information may improve the test execution 
+              # planning algorithm performance.  
+              "duration": "2s",
+                
+              # Timeout information may also support efficient
+              # planning.
+              "timeout": "16s",
+            },
+            
+            
+        }  
+    ],
+    
+    # Additional suites the extension wants to advertise / participate in.
+    "suites": [
+        {
+            # Here, the extension is advertising a new suite, unknown
+            # to the origin framework until discovered through the
+            # binary. 
+            "name": "fips/conformance",
+
+            "parents": [
+
+                # This suite can be run separately or will be included
+                # automatically in other suites known to origin.
+                # Here, the extension is informing origin that the suite
+                # is a sub-suite of openshift/conformance. All information
+                # about a suite is additive across extension binaries.
+                # For example, multiple binaries can advertise the same
+                # suite -- if they advertise different parents, then
+                # origin will treat the suite as a subset of each 
+                # identified parent.
+                "openshift/conformance"
+            ],

```

Comment:
How are parents and children joined together?

In this example, is the suite named "openshift/conformance/fips/conformance"?

Reply:
As imagined here, `fips/conformance` is a suite that can be executed by origin in isolation -- the specified name is fully qualified. 
However, because it is claiming `openshift/conformance` as a parent, if origin runs `openshift/conformance`, tests from this suite definition will be included.

Reply:
openshift-tests-extension (OTE) as written allows contributing tests to multiple suites - if we can do that, does we need the idea of parents?